internal/api/api.go

```diff
@@ -13,7 +13,9 @@ import (
 	"github.com/fabric8-services/fabric8-jenkins-idler/internal/model"
 	"github.com/fabric8-services/fabric8-jenkins-idler/internal/openshift"
 	"github.com/fabric8-services/fabric8-jenkins-idler/internal/openshift/client"
+	ss "github.com/fabric8-services/fabric8-jenkins-idler/internal/status"
```

Comment:
This doesn't seems to be consistent. There's ss, userStatus and status.

Reply:
Done